filesmanager/settings/production.py

```diff
@@ -6,8 +6,14 @@
 """
 
 
-def plugin_settings(settings):  # pylint: disable=unused-argument
+def plugin_settings(settings):
     """
     Set of plugin settings used by the Open Edx platform.
     More info: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/plugins/README.rst
     """
+    settings.FILES_MANAGER_EVENT_ROUTING_BACKEND = getattr(
```

Comment:
this is deprecated, isn't it ? I think nobody uses the ENV_TOKENS approach

Reply:
I'm not sure if it is deprecated, we have used it in other xblocks and plugins. how else would it be done?

Reply:
I mean this approach is still valid before tutor, if this xblock is supported in a different tutor infrastructure  this is still valid, however is not common to change the default settings and this specific setting is related to a library that probably nobody will change

Reply:
Do you think it is not necessary to include the settings from here?